build.gradle

```diff
@@ -0,0 +1,351 @@
+buildscript {
+    repositories {
+        maven {
+            url = uri("https://projects.itemis.de/nexus/content/repositories/mbeddr")
+        }
+        mavenCentral()
+        mavenLocal()
+    }
+    dependencies {
+        classpath "de.itemis.mps:mps-gradle-plugin:$mps_gradle_plugin_version"
+    }
+}
+plugins {
+    id "com.github.node-gradle.node" version "2.2.3"
+    id "java"
+    id "generate-models"
+}
+
+group 'com.discpl'
```

Comment:
This should be `org.disicpl` Note that you're also missing the second `i`, which is also missing in other places, could you fix that as well? :innocent: (Just wanna make sure we get this right, because namespacing changes after the fact can sometimes be painful)

Reply:
`org.discipl` bedoel je toch en volgens mij zit je naar een oude commit te kijken.

Reply:
Apologies. You're right about the old comit and spelling discipl is hard:P. The latter point still stands though, `discpl` is still present in the current state, right? https://github.com/discipl/flinteditor-mps/pull/1/files#diff-cc6bc19a7a530133881a03b5c41cba02

Reply:
Should be resolved now